lib/Doctrine/Migrations/AbstractMigration.php

```diff
@@ -155,6 +163,11 @@ public function getSql() : array
         return $this->plannedSql;
     }
 
+    /**
+     * Write some debug information to the console.
+     *
+     * Debug information is written with debug().
```

Comment:
:thinking: I mean it should, but it's actually written with `notice()`, so that comment is just misleading